tmt/steps/__init__.py

```diff
@@ -1018,7 +1018,7 @@ def go_prolog(self, logger: tmt.log.Logger) -> None:
         # Include order in verbose mode
         logger.verbose('order', str(self.order), 'magenta', level=3)
 
-    def requires(self) -> List['tmt.base.Require']:
+    def requires(self) -> List['tmt.base.Dependency']:
         """ All requirements of the plugin on the guest """
```

Comment:
Throughout the codebase, there are a bunch of what feels like leftovers. This method is called `requires`, but it returns `List[Dependency]`, and the docstring mentions *All requirements*. Would it be feasible to somehow unify the wording? I feel like this might add some confusion later.

Reply:
Yeah, I didn't want to touch the method names too much. I don't mind changing `requires` to `dependencies`, but that means mean fallout, bigger patch, and harder to review. I prefer doing it in smaller steps, renaming structures, and renaming (internal) APIs. Making it all consistent is definitely the desired state, and I did plan to add more PRs later. Some `require` and `recommend` strings will remain "forever", as they refer to tmt metadata keys, but yep, the name of this method is something that might change.